contracts/BaseStrategy.sol

```diff
@@ -391,172 +327,51 @@ abstract contract BaseStrategy {
-     * on protected functions in the Strategy.
-     */
     function governance() internal view returns (address) {
         return vault.governance();
     }
 
-    /**
-     * @notice
-     *  Provide an accurate conversion from `_amtInWei` (denominated in wei)
-     *  to `want` (using the native decimal characteristics of `want`).
-     * @dev
-     *  Care must be taken when working with decimals to assure that the conversion
-     *  is compatible. As an example:
-     *
-     *      given 1e17 wei (0.1 ETH) as input, and want is USDC (6 decimals),
-     *      with USDC/ETH = 1800, this should give back 1800000000 (180 USDC)
-     *
-     * @param _amtInWei The amount (in wei/1e-18 ETH) to convert to `want`
-     * @return The amount in `want` of `_amtInEth` converted to `want`
-     **/
+    function management() internal view returns (address) {
+        return vault.management();
+    }
+
     function ethToWant(uint256 _amtInWei) public virtual view returns (uint256);
 
-    /**
-     * @notice
-     *  Provide an accurate estimate for the total amount of assets
-     *  (principle + return) that this Strategy is currently managing,
-     *  denominated in terms of `want` tokens.
-     *
-     *  This total should be "realizable" e.g. the total value that could
-     *  *actually* be obtained from this Strategy if it were to divest its
-     *  entire position based on current on-chain conditions.
-     * @dev
-     *  Care must be taken in using this function, since it relies on external
-     *  systems, which could be manipulated by the attacker to give an inflated
-     *  (or reduced) value produced by this function, based on current on-chain
-     *  conditions (e.g. this function is possible to influence through
-     *  flashloan attacks, oracle manipulations, or other DeFi attack
-     *  mechanisms).
-     *
-     *  It is up to governance to use this function to correctly order this
-     *  Strategy relative to its peers in the withdrawal queue to minimize
-     *  losses for the Vault based on sudden withdrawals. This value should be
-     *  higher than the total debt of the Strategy and higher than its expected
-     *  value to be "safe".
-     * @return The estimated total assets in this Strategy.
-     */
     function estimatedTotalAssets() public virtual view returns (uint256);
 
-    /*
-     * @notice
-     *  Provide an indication of whether this strategy is currently "active"
-     *  in that it is managing an active position, or will manage a position in
-     *  the future. This should correlate to `harvest()` activity, so that Harvest
-     *  events can be tracked externally by indexing agents.
-     * @return True if the strategy is actively managing a position.
-     */
     function isActive() public view returns (bool) {
         return vault.strategies(address(this)).debtRatio > 0 || estimatedTotalAssets() > 0;
     }
 
-    /**
```

Comment:
is the linter removing all these docs in the methods?

Reply:
yeah wtf is up with this?

Reply:
running `yarn lint` removes the doc.